hoard/sources/whoas.py

```diff
@@ -184,6 +184,8 @@ def create_from_whoas_dim_xml(data: str, client: OAIClient) -> Dataset:
             kwargs["termsOfUse"] = field.text
 
     kwargs["subjects"] = ["Earth and Environmental Sciences"]
+    if "description" not in kwargs:
+        kwargs["description"] = kwargs["title"]
```

Comment:
This won't work because the description field in our Dataset model isn't a string like title is. The string will need to be passed to a Description instance as we do above in the code where a description is provided in the metadata. I think the new validators code would have caught this but just wasn't merged yet :)

Reply:
Ugh, yes, the validators would have set me straight. Fixed